src/styles/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
```

Comment:
Combine selectors with the same styles into one selector to follow the DRY principle. You can use SCSS nesting to achieve this, which will make your code cleaner and easier to maintain. For example:
.stars {
  &--1 :nth-child(-n+1),
  &--2 :nth-child(-n+2),
  &--3 :nth-child(-n+3),
  &--4 :nth-child(-n+4),
  &--5 :nth-child(-n+5) {
    background-image: url('../images/star-active.svg');
  }
}